src/visitor/mysql.rs

```diff
@@ -251,6 +260,18 @@ impl<'a> Visitor<'a> for Mysql<'a> {
         Ok(())
     }
 
+    fn visit_upsert(&mut self, update: Update<'a>) -> visitor::Result {
+        if update.conditions.is_some() {
+            //TODO should this rather be an error or do we just ignore?
+            panic!("cannot use a where clause for a mysql upsert");
+        }
```

Comment:
Hmm, won't this limitation not fit with our existing upsert API? For instance, how would you express the following query given that MySQL doesn't support a `where` clause?

```
upsertOneUser({
  where: { email: "bob@prisma.io" },
  create: { id: 1, email: "alice@prisma.io", name: "alice" },
  data: { email: "alice@prisma.io" }
})
```